apps/tsilhqotin-language-hub/src/Components/Widgets/Navbar/Navbar.tsx

```diff
@@ -26,6 +26,7 @@ const Navbar = () => {
             <Typography component={'span'} sx={{ margin: '15px', padding: '15px' }}>
                 <NavMenu className="navLinks">
                     <NavLink to="/About">About</NavLink>
+                    <NavLink to="/Radio">Radio</NavLink>
```

Comment:
How about `104.5` or something with `104.5 in it` for the text on this link?

Reply:
Fixed.